README.md

```diff
@@ -2,17 +2,24 @@
 
 ## Table of Contents
 
-- [Features](#features)
+- [Preparations](#preparations)
 - [Install](#install)
 - [Usage](#usage)
 <!-- - [License](#license) -->
 
-## Features
-
 ### Stack
 
 - NodeJS + [Typescript](https://github.com/microsoft/TypeScript)
 
+## Preparations
```

Comment:
We can keep this for now, but can you please create an issue to remove mongo only stuff and do everything in docker-compose